src/emmetHelper.ts

```diff
@@ -490,8 +490,8 @@ export function isAbbreviationValid(syntax: string, abbreviation: string): boole
 		return parseInt(multipleMatch[1], 10) <= 100
 	}
 	// Its common for users to type (sometextinsidebrackets), this should not be treated as an abbreviation
-	// Grouping in abbreviation is valid only if preceeded/succeeded with one of the symbols for nesting, sibling, repeater or climb up
-	if (!/\(.*\)[>\+\*\^]/.test(abbreviation) && !/[>\+\*\^]\(.*\)/.test(abbreviation) && /\(/.test(abbreviation) && /\)/.test(abbreviation)) {
+	// Grouping in abbreviation is valid only if it's inside a text node or preceeded/succeeded with one of the symbols for nesting, sibling, repeater or climb up
+	if ((/\(/.test(abbreviation) || /\)/.test(abbreviation)) && !/\{[^\}\{]*[\(\)]+[^\}\{]*\}(?:[>\+\*\^]|$)/.test(abbreviation) && !/\(.*\)[>\+\*\^]/.test(abbreviation) && !/[>\+\*\^]\(.*\)/.test(abbreviation)) {
```

Comment:
If it's not obvious then the `(?:[>\+\*\^]|$)` is to invalidate abbreviations like `div{ a }( b ){ c }`.